gateway/gateway.go

```diff
@@ -303,7 +303,7 @@ func (gw *Gateway) handleMessage(msg config.Message, dest *bridge.Bridge) []*BrM
 		msg.Avatar = gw.modifyAvatar(origmsg, dest)
 		msg.Username = gw.modifyUsername(origmsg, dest)
 		msg.ID = ""
-		msg.ParentID = ""
```

Comment:
Wait, I accepted this, but I'm pretty sure this needs to be "reset" to empty each time, or it persists between for gateways that have more than two bridges. Or is there something I'm missing?